whipper/common/encode.py

```diff
@@ -89,3 +90,71 @@ def _tag(self):
         w.save()
 
         self.stop()
+
+
+class EmbedPictureTask(task.Task):
+    description = 'Embed picture to FLAC'
+
+    def __init__(self, track_path, coverart_path):
```

Comment:
Let’s be consistent:
```suggestion
    def __init__(self, track_path, cover_art_path):
```

(and `s/coverart_path/cover_art_path/g`)